benchmark/infrastructure/BenchmarkMeasurementContainer.cpp

```diff
@@ -364,4 +370,36 @@ std::ostream& operator<<(std::ostream& os, const ResultGroup& resultGroup) {
   return streamInserterOperatorImplementation(os, resultGroup);
 }
 
+// ____________________________________________________________________________
+template <ad_utility::isTypeAnyOf<ResultEntry, ResultTable> T>
+requires std::same_as<std::decay_t<T>, T>
+void ResultGroup::deleteEntryImpl(T& entry) {
+  // The vector, that holds our entries.
+  auto& vec{std::invoke([this]() -> auto& {
```

Comment:
better:
auto& vec = [...](...) {...}(); (immediately invoked lambda).

Reply:
I don't quite follow. It is already an immediately invoked lambda.

Reply:
```
auto bla = []() {return 42;}();

auto blubb{std::invoke([](){return 42;}};
```
I find the first version  easier to read: + clang-format gives the better formatting.

Reply:
Done, see [fb27100](https://github.com/ad-freiburg/qlever/pull/1207/commits/fb271000d71ca7a0f49c7c99c132b2f4e1f66fd7).